src/main/java/com/webperside/courseerpbackend/controller/CountryController.java

```diff
@@ -0,0 +1,26 @@
+package com.webperside.courseerpbackend.controller;
+
+import com.webperside.courseerpbackend.models.mybatis.country.Country;
+import com.webperside.courseerpbackend.services.country.CountryServiceImpl;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/v1/countries")
+public class CountryController {
+
+    private final CountryServiceImpl countryService;
+
+    @PostMapping
+    public void insert(@RequestBody Country country){
```

Comment:
don't use Mybatis model for accepting payload
create new class for that, f.e CountryPayload